.github/workflows/publish-doxygen.yml

```diff
@@ -0,0 +1,15 @@
+name: Doxygen GitHub Pages Deploy Action
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
pushes webpage to `gh-pages`. In repo, setting this branch can be enabled to serve Github pages.

Alternative would be to publish the documentation in some other location (e.g. Khronous owned) or just leave it for users to build locally.